Sources/TuistKit/ProjectEditor/ProjectEditor.swift

```diff
@@ -93,6 +92,8 @@ final class ProjectEditor: ProjectEditing {
         let projectDescriptionPath = try resourceLocator.projectDescription()
         let projectManifests = manifestFilesLocator.locateProjectManifests(at: editingPath)
         let configPath = manifestFilesLocator.locateConfig(at: editingPath)
+        let cacheDirectory = cacheDirectoryProviderFactory.cacheDirectories(config: nil)
+        let projectDescriptionHelpersBuilder = ProjectDescriptionHelpersBuilder(cacheDirectory: cacheDirectory.projectDescriptionHelpersCacheDirectory)
```

Comment:
I don't think this is correct as we will use the real implementation of helpers builders in unit tests

Reply:
fixed